PhysicsTools/PatAlgos/plugins/CaloJetSlimmer.cc

```diff
@@ -0,0 +1,63 @@
+
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "DataFormats/Common/interface/RefVector.h"
+
+#include "CommonTools/UtilAlgos/interface/StringCutObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/ObjectSelector.h"
+#include "CommonTools/UtilAlgos/interface/SingleElementCollectionSelector.h"
+
+#include "DataFormats/JetReco/interface/CaloJet.h"
+#include "DataFormats/PatCandidates/interface/Jet.h"
+#include "DataFormats/Candidate/interface/Candidate.h"
+
+#include <vector>
+
+class CaloJetSlimmer : public edm::stream::EDProducer<> {
+
+public:
+
+CaloJetSlimmer( edm::ParameterSet const & params ):
+    srcToken_(consumes<edm::View<reco::CaloJet> >( params.getParameter<edm::InputTag>("src") )),
+    cut_( params.getParameter<std::string>("cut") ),
+    selector_( cut_ )
+    {
+        produces< reco::CaloJetCollection> ();//("slimmedCaloJets");
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant.